// Copyright (c) 2018 The Open-Transactions developers
// This Source Code Form is subject to the terms of the Mozilla Public
// License, v. 2.0. If a copy of the MPL was not distributed with this
// file, You can obtain one at http://mozilla.org/MPL/2.0/.

syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTBlockchainBlockLocalData";
option optimize_for = LITE_RUNTIME;

message BlockchainBlockLocalData
{
    optional uint32 version = 1;
    optional int64 height = 2;
    optional uint32 status = 3;
    optional uint32 inherit_status = 4;
    optional string work = 5;
    optional string inherit_work = 6;
}
